### ISSUE TITLE
Expose average price this month 

### ISSUE THREAD

**Issue by slasktrat** [frodeheg/no.sparegris#125]
The app already keeps track of the monthly average price. It would be great if current month's value could be made available to use in flow cards, as that info would be relevant to perform actions based on "strømstøtte". Maybe also a dedicated price level in the app to configure custom deltas when negative price after "strømstøtte".

**Comment by frodeheg** [comment]
Note that the average value I keep track of include grid cost, surecharge and VAT, so you would have to do some calculations on it to make it usable for such a purpose...

**Comment by slasktrat** [comment]
Good point, but maybe the app also could keep track of the price alone?
